blueflood-core/src/main/java/com/rackspacecloud/blueflood/inputs/formats/JSONMetricsContainer.java

```diff
@@ -87,7 +92,7 @@ public boolean areDelayedMetricsPresent() {
     public static class JSONMetric {
         private String metricName;
         private Object metricValue;
-        private long collectionTime;
+        private Long collectionTime = null;
```

Comment:
Do you have to change this to "Long"? Instead of ``collectionTime != null``, cant you do collectionTime > 0?